eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/AbstractMutableBiMap.java

```diff
@@ -543,6 +545,10 @@ public MutableShortBag collectShort(ShortFunction<? super V> shortFunction)
         return this.delegate.collectIf(predicate, function);
     }
 
+    /**
+     * @deprecated in 7.1. Use {@link OrderedIterable#zip(Iterable)} instead.
+     */
+    @Deprecated
```

Comment:
It looks odd to be included in the same commit. Could this be in a separate commit? Version number should be 8.0 now that 7.1 is already released. Also the link should be `zipWithIndex(Iterable)` not `zip(Iterable)` I think.